### PR TITLE
disable agg function in ngql's yield clause & where clause

### PR DESCRIPTION
**Pull request by nevermore3**
#### What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Review by Shylock-Hg — DISMISSED** [review]

**Comment by Shylock-Hg** [comment]
I think you could remove related checking code in validator.

**Comment by nevermore3** [comment]
> I think you could remove related checking code in validator.

done

File: src/graph/validator/FetchEdgesValidator.cpp [hunk 1]
```diff
@@ -152,8 +152,7 @@ Status FetchEdgesValidator::validateYield(const YieldClause *yield) {
   auto pool = qctx_->objPool();
   auto *newCols = pool->add(new YieldColumns());
   for (auto col : yield->columns()) {
-    if (ExpressionUtils::hasAny(col->expr(),
```
Comment (Shylock-Hg) [inline src/graph/validator/FetchEdgesValidator.cpp:155]: Now support path build expression?

Comment (nevermore3) [reply src/graph/validator/FetchEdgesValidator.cpp:155]: yes, pathbuildExpression Is used internally, and the user cannot construct it

**Review by Shylock-Hg — DISMISSED** [review]

**Review by Shylock-Hg — DISMISSED** [review]

File: tests/tck/features/aggregate/Agg.feature [hunk 2]
```diff
@@ -691,12 +691,12 @@ Feature: Basic Aggregate and GroupBy
       """
       GO FROM "Tim Duncan" OVER like YIELD count(*)
       """
-    Then a SemanticError should be raised at runtime: `count(*)' is not support in go sentence.
+    Then a SyntaxError should be raised at runtime: Invalid use of aggregating function in yield clause. near `count(*)'
```
Comment (jievince) [inline tests/tck/features/aggregate/Agg.feature:694]: Why disable this usage?

Comment (nevermore3) [reply tests/tck/features/aggregate/Agg.feature:694]: This  was not allowed before, and using  agg  funciton in ngql 's  yield clause  is meaningless

**Review by jievince — DISMISSED** [review]

**Review by jievince — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**